pyproject.toml

```diff
@@ -0,0 +1,5 @@
+# pyproject.toml
+[build-system]
+requires = ["setuptools>=41.2.0",
```

Comment:
This version of `setuptools` is too strict.  That looks like the latest one, but it will work with earlier versions as well.  You can probably not specify a version for `setuptools` or `setuptools_scm` and it will be fine.

Reply:
Removed version number for `setuptools_scm` and lowered the version number for `setuptools` to match that in the `jwst` repo (38.2.5)